python/hopsworks/project.py

```diff
@@ -101,7 +102,7 @@ def created(self):
         """Timestamp when the project was created"""
         return self._created
 
-    def get_feature_store(self, name: str = None):
+    def get_feature_store(self, name: str = None) -> feature_store.FeatureStore:
```

Comment:
Does it not need also the module here? `hsfs.feature_store.FeatureStore`
https://docs.hopsworks.ai/feature-store-api/latest/generated/api/feature_store_api/

Reply:
It is imported above from hsfs import feature_store